wait_for_copr/__init__.py

```diff
@@ -55,29 +55,22 @@ def wait_for_copr(owner, project, max_tries, interval, dependency, release):
     )
 
     for _ in range(max_tries):
-        copr_package_proxy = client.package_proxy
-
-        if not copr_package_proxy:
-            # Might be caused by a network issue.
-            click.echo("Issue with initiating Copr package proxy. Retrying.")
-            time.sleep(interval)
-            continue
```

Comment:
with `build_proxy` this is not an issue?

Reply:
I should have made a comment about that in the commit message. My bad. We just discussed with Pavel that he's curious to reproduce this and asked me to get rid of that handling..;)

Reply:
@lbarcziova yeah the point was to fix the issue in Copr, if there (still) is one.  Not sure I got your question "with build_proxy this is not an issue?" - is there any other proxy suffering from similar problems?

Reply:
I thought since previously the implementation used `package_proxy` and we looked for this problem in the code, that since with switching to `build_proxy` this was removed, it's related. But I get it now, thanks for the explanation!

Reply:
aaaah, now it makes sense, I did not even notice the proxy change :-) thanks